lib/useOnyx.ts

```diff
@@ -278,7 +272,7 @@ function useOnyx<TKey extends OnyxKey, TReturnValue = OnyxValue<TKey>>(key: TKey
         checkEvictableKey();
     }, [checkEvictableKey]);
 
-    const result = useSyncExternalStore<UseOnyxResult<TKey, TReturnValue>>(subscribe, getSnapshot);
+    const result = useSyncExternalStore(subscribe, getSnapshot);
```

Comment:
Why was this removed?

Reply:
Do we need to define it explicitly?

Reply:
It's to ensure `getSnapshot` **has** to return this value `UseOnyxResult<TReturnValue | undefined>` as a code protection measure.